src/status_im2/contexts/quo_preview/keycard/keycard.cljs

```diff
@@ -0,0 +1,37 @@
+(ns status-im2.contexts.quo-preview.keycard.keycard
+  (:require [status-im2.contexts.quo-preview.preview :as preview]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [quo2.components.keycard.view :as quo2]))
```

Comment:
`quo` previews should import from `quo2.core`, since that's the public API. Maybe we haven't yet migrated all previews to the correct imports, so maybe you got this pattern from those namespaces.